2021/36xxx/CVE-2021-3647.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Expected location (2021/3xxx/CVE-2021-3647.json) of file does not match actual location (2021/36xxx/CVE-2021-3647.json)